integration_tests/README.md

```diff
@@ -149,7 +149,7 @@ Next you can start to run the tests.
 
 ```scala
 durations.run(new com.nvidia.spark.rapids.JoinsSuite)
-...
+// ...
```

Comment:
is this on purpose?

Reply:
@abellina: Good eye! Yes, this was intentional. The IDE flags the `...` as bad Scala, because it's in a `scala` block.

Reply:
Ah ok. Perhaps just removing the line would be fine? It prints the line as `// ...`, according to github, and it doesn't seem like it adds much.

Reply:
Gotcha. Will remove.

Reply:
Fixed. Pushed new version.